src/interactive/azext_interactive/azclishell/app.py

```diff
@@ -859,7 +859,17 @@ def cli_execute(self, cmd):
                 self.threads.append(thread)
                 result = None
             else:
-                result = invocation.execute(args)
+                try:
+                    result = invocation.execute(args)
+                    # Prevent users from exiting the entire az interactive by using Ctrl+C during command execution
+                except KeyboardInterrupt:
+                    result = None
+                    self.last_exit_code = 1
+                except SystemExit as ex:
+                    # prevent errors caused by uncompleted command loading
+                    if ex.code == 2 and self.command_table_thread.is_alive():
+                        print_styled_text([(Style.ERROR, "Command loading is not complete, please wait...")])
```

Comment:
Do we need to consider printing error logs for other cases of `SystemExit` other than `uncompleted command loading`?

Reply:
When the command is fully loaded, other error factors that cause `SystemExit` can be correctly caught and the correct error message will be output by the Azure CLI, requiring no additional processing by us in `az interactive`.

Reply:
OK, got it